config-path.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const userHome = require('user-home');
+
+const env = process.env;
+const name = 'js-v8flags';
+
+function macos () {
+	const library = path.join(userHome, 'Library');
+	return path.join(library, 'Caches', name);
+}
+
+function windows () {
+	const appData = env.LOCALAPPDATA || path.join(userHome, 'AppData', 'Local');
+	return path.join(appData, name, 'Cache');
+}
+
+// https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+function linux () {
+	const username = path.basename(userHome);
+	return path.join(env.XDG_CACHE_HOME || path.join(userHome, '.cache'), name);
+}
+
+if (process.platform === 'darwin') {
```

Comment:
No problem, I'm not sure as both ways of their pros and cons. Think as it is now it's very specific to this package so I would leave it as it is. When this package drops support for < Node 4 ideally we would swap it with the `env-paths` package, by then the 'new ES import' will be in place too so it has more significance to be able to statically analyze it.
:thinking: